src/pages/components/GridSelectorAndLabel.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Label } from 'src/pages/components/Label'
+import Grid from '@mui/material/Unstable_Grid2'
+
+export function GridSelectorAndLabel(props: { label: string; children: React.ReactNode }) {
+  return (
+    <>
+      <Grid xs={4}>
```

Comment:
You should make the "xs" configurable so you could use this component in other places (see comments in "RealtimeMapPage")